cpp/src/gandiva/precompiled/string_ops.cc

```diff
@@ -1246,6 +1246,69 @@ const char* convert_fromUTF8_binary(gdv_int64 context, const char* bin_in, gdv_i
   return ret;
 }
 
+#define CONVERT_TO_FN(TYPE)                                                              \
```

Comment:
may can also add functions for bigendian outputs

Reply:
I add a function that convert the value in the reversed way from the native architecture. If the arrow is using little endian, it will convert using big-endian and if is using big-endian, it will convert using little-endian